.vscode/launch.json

```diff
@@ -10,7 +19,7 @@
             "windows": {
               "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron.cmd"
             },
-            "runtimeArgs": ["./dist/tuxedo-control-center"],
+            "runtimeArgs": ["./dist/tuxedo-control-center", "--debug"],
```

Comment:
Is this just for executions in VSCode or for release as well?

Reply:
`--debug` is only passed when you run the `Debug Main Process` target from vscode. It would possible to pass to a release build as well. I'm not sure how to avoid that though.

It opens the dev tools / inspector immediately as the application launches which I found useful, especially for placing breakpoints and stepping through things.